src/arduino/components/potentiometer.h

```diff
@@ -0,0 +1,14 @@
+class Potentiometer
```

Comment:
Again no longer handled on Arduino, a rasp pi version of this is needed.

Gonna cc @thatnoobles  on these btw cause I don't know what came in from his PR

Reply:
Removed references to potentiometers in the C++ code since the Arduino no longer knows about any of that. I've updated the RPi class to reflect that it is going to be receiving potentiometer data from the ADC now.